tools/bpf2c/bpf2c_driver.c

```diff
@@ -143,7 +143,7 @@ _bpf2c_npi_client_attach_provider(
     // As per MSDN, client dispatch can be NULL, but SAL does not allow it.
     // https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/netioddk/nf-netioddk-nmrclientattachprovider
```

Comment:
I don't think this comment is relevant any more now that the third argument below is non-NULL.